subsys/bluetooth/controller/Kconfig

```diff
@@ -379,7 +379,8 @@ config BT_CTLR_DATA_LENGTH
 config BT_CTLR_DATA_LENGTH_MAX
 	int "Maximum data length supported"
 	depends on BT_CTLR_DATA_LENGTH
-	default 27
+	default BT_BUF_ACL_RX_SIZE if BT_BUF_ACL_RX_SIZE < 251
+	default 251
```

Comment:
`BT_BUF_ACL_RX_SIZE` defaults to 69 bytes for upstream samples like peripheral_hr, where as `BT_BUF_ACL_TX_SIZE` is 27 bytes. The default should remain 27 bytes, and only samples/application needing large PDU sizes should explicitly set as per requirement.

Reply:
@cvinayak , could you elaborate why you do not want to change this? 

If `BT_CTLR_DATA_LENGTH_MAX` is set to 27, there is really no point in enabling the feature. Then I would rather suggest setting `CONFIG_BT_CTLR_DATA_LENGTH=n` in these samples.